server_plugin/server.py

```diff
@@ -254,7 +272,7 @@ def _get_state(vca_client):
         for connection in nw_connections}
 
     for connection in nw_connections:
-        if connection['network_name'] == management_network_name:
+        if ctx.node.properties.get('use_external_resource') or connection['network_name'] == management_network_name:
```

Comment:
not sure i understand this part. if i use an external resources, why are all nw_connections necessarily the management network?